docs/configuration/module/telemetry.mdx

```diff
@@ -133,6 +148,10 @@ All telemetry module config options are available in the python CLI. Example com
 |   telemetry.environment_update_interval   | `integer` (seconds) | Default `0` is 30 minutes(`1800` seconds). |
 |   telemetry.air_quality_enabled           |   `true`, `false`   |                `false`                    |
 |   telemetry.air_quality_interval          | `integer` (seconds) | Default `0` is 30 minutes(`1800` seconds). |
+|   telemetry.health_measurement_enabled    |   `true`, `false`   |                `false`                    |
+|   telemetry.health_update_in              | `integer` (seconds) | Default `0` is 30 minutes(`1800` seconds). |
```

Comment:
I believe there is a type here: `telemetry.health_update_in` -> `health_update_interval `

Reply:
Thanks!